plugins/PrepareStage/PrepareMenu.qml

```diff
@@ -34,6 +33,13 @@ Item
             action: Cura.Actions.open
         }
 
+        Item
+        {
+            id: spacing
+            width: UM.Theme.getSize("default_margin").width
+            height: parent.height
+        }
```

Comment:
It's more efficient to give MachineAndConfigurationSelector a margin, I'm thinking.